components/movementsensor/gpsrtk/ntrip.go

```diff
@@ -1,35 +1,13 @@
 package gpsrtk
 
 import (
-	"context"
 	"fmt"
 	"io"
-	"sync"
 
 	"github.com/de-bkg/gognss/pkg/ntrip"
 	"github.com/edaniels/golog"
-	"github.com/go-gnss/rtcm/rtcm3"
-	"github.com/pkg/errors"
-	"go.viam.com/utils"
-
-	"go.viam.com/rdk/components/movementsensor"
 )
 
-type ntripCorrectionSource struct {
-	logger      golog.Logger
-	ntripStatus bool
-
-	cancelCtx               context.Context
-	cancelFunc              func()
-	activeBackgroundWorkers sync.WaitGroup
-
-	mu               sync.Mutex
-	correctionReader io.ReadCloser
-	info             *NtripInfo
-
-	err movementsensor.LastError
```

Comment:
we should reorganize this while we are at it. Let's move all the RTK station stuff out of gpsrtk. Make gpsrtk look a lot cleaner.

Reply:
Can you be more specific about what you mean here? I think the rtk_station is already seperate from gpsrtk.

Reply:
Oh yeah definitely might make sense to move them to separate packages, I think that can be addressed in a separate PR once the scope doc is approved since we are likely changing base_station to the sensor api and moving to viam labs.

Reply:
It is okay to change them before the scope is approved as long as the model names and config are retained and the drivers do the same thing as they do now. But happy to wait or do that in a separate PR. The work for moving them is nearly equivalent.